### PR TITLE
Added beaglebone black revA6A support

### PR DESCRIPTION
**Pull request by rbalsan1**

### CODE REVIEW

File: adafruit_platformdetect/constants/boards.py [hunk 1]
```diff
@@ -324,7 +324,7 @@
 # BeagleBone eeprom board ids from:
 #   https://github.com/beagleboard/image-builder
 # Thanks to zmatt on freenode #beagle for pointers.
-_BEAGLEBONE_BOARD_IDS = {
+_BEAGLEBONE_BOSARD_ID = {
```
Comment (makermelissa) [inline adafruit_platformdetect/constants/boards.py:327]: Hi, it looks like you accidentally changed the name here?

**Review by makermelissa — CHANGES_REQUESTED** [review]
This looks like an easy fix

**Review by rbalsan1 — COMMENTED** [review]
Yep I did

File: adafruit_platformdetect/constants/boards.py (hunk 1 above)
Comment (rbalsan1) [reply adafruit_platformdetect/constants/boards.py:327]: Yes I think its Fixed

Comment (ladyada) [reply adafruit_platformdetect/constants/boards.py:327]: it is not fixed

**Comment by makermelissa** [comment]
Could you push the changes you made?

File: adafruit_platformdetect/constants/boards.py (hunk 1 above)
Comment (rbalsan1) [reply adafruit_platformdetect/constants/boards.py]: Thanks for letting me learn how to correct my corrections.  I think we are good now

**Review by makermelissa — APPROVED** [review]
Looks good. Thank you.

**Pull request merged**